notebooks/hf_llama_generation_example.ipynb

```diff
@@ -20,15 +20,15 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 2,
```

Comment:
change the model path to something more generic and less your specific desktop path